native_client/java/libdeepspeech/src/androidTest/java/org/mozilla/deepspeech/libdeepspeech/test/BasicTest.java

```diff
@@ -149,4 +151,19 @@ public void loadDeepSpeech_sttWithMetadata_withLM() {
         assertEquals("she had your dark suit in greasy wash water all year", decoded);
         m.freeModel();
     }
+
+    @Test
+    public void loadDeepSpeech_HotWord() {
```

Comment:
Please add a second function for makign another test:
 - create a model
 - DONT call `enableExternalScorer`
 - try to add a HotWord

And assert that we throw an error `DS_ERR_SCORER_NOT_ENABLED` `0x2004`

Reply:
sure @lissyx I will fix it up